compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/intrinsics/KClassJavaObjectTypeProperty.kt

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Please fix inspections and reformat this file and `KClassJavaPrimitiveTypeProperty.kt`